projects/qcms/build.sh

```diff
@@ -15,24 +15,9 @@
 #
 ################################################################################
 
-# moz.build
-$CC $CFLAGS -c \
-  -DNDEBUG -mavx -msse -msse2 -Wno-missing-field-initializers \
-  chain.c \
-  iccread.c \
-  matrix.c \
-  transform.cpp \
-  transform-avx.cpp \
-  transform-sse1.cpp \
-  transform-sse2.cpp \
-  transform_util.c
+cargo fuzz build -O
 
-$CXX $CXXFLAGS -std=c++11 \
-  -I. *.o \
-  -DBUILD_FOR_OSSFUZZ \
-  $LIB_FUZZING_ENGINE \
-  fuzztest/qcms_fuzzer.cpp -o $OUT/fuzz
-
-find fuzztest -iname "*.icc" \
+find fuzz -iname "*.icc" \
   -type f -exec zip -qju $OUT/fuzz_seed_corpus.zip "{}" \;
-cp fuzztest/qcms_fuzzer.dict $OUT/fuzz.dict
+cp fuzz/qcms_fuzzer.dict $OUT/fuzz.dict
+cp fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_target_qcms $OUT/
```

Comment:
Does this need to be $OUT/fuzz

Reply:
https://google.github.io/oss-fuzz/getting-started/new-project-guide/rust-lang/ suggests that '$OUT/' is sufficient.

Reply:
Ah. That makes sense.

Reply:
And I pushed a changed version before seeing your commit